### DISCUSSION TITLE
Truncate long values in Model __repr__

### DISCUSSION THREAD

**Discussion by johtso** [pydantic/pydantic#2756] [Feature Request]
I'm working with some models that have some rather long strings as values, and the resulting `__repr__` ends up huge.

Would it make sense to truncate it in the `__repr__` output?

**Comment by ileixe** [comment]
Yes, I also need a similar functionality truncating the long bytes field. I saw there is an incoming feature to hide a specific field via `repr=False` though, it's nicer to have a field there not to completely hide it.
